script/settings.json

```diff
@@ -0,0 +1,20 @@
+{
+  "folders": [
+    "../ci",
+    "../automation"
+  ],
+  "enabledActions": [
+    "actions/checkout",
```

Comment:
Do you think it'd be better to pull these "live" by iterating over the folder in the ghes-actions repo? (https://github.com/github/ghes-actions/tree/master/config/actions)

Reply:
That could be a future improvement.